lib/api/core/plugins/pluginContext.js

```diff
@@ -78,24 +77,17 @@ class PluginContext {
       const
         kuzzleSdk = new KuzzleSDK(new FunnelProtocol(kuzzle.funnel)),
         // uppercase are forbidden by ES
-        pluginIndex = `plugin:${pluginName}`.toLowerCase(),
-        pluginIndexEngine = new IndexEngine(
-          kuzzle,
+        pluginIndex = `plugin-${pluginName}`.toLowerCase(),
+        pluginIndexStorage = new IndexStorage(
           pluginIndex,
-          kuzzle.services.internalStorage);
-
-      // Set the bootstraper class to initialize plugin index
-      pluginIndexEngine.indexBootstap = new PluginIndexBootstrap(
-        kuzzle,
-        pluginName,
-        pluginIndexEngine);
+          kuzzle.storageEngine.internal);
 
       Object.defineProperty(this.accessors, 'storage', {
         enumerable: true,
         get: () => {
           return {
-            bootstrap: () => pluginIndexEngine.init(),
-            createCollection: (...args) => pluginIndexEngine.createCollection(...args)
+            bootstrap: (collections) => pluginIndexStorage.init(collections),
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS3524)